src/NzbDrone.Core/MediaFiles/EpisodeImport/DetectSample.cs

```diff
@@ -8,7 +8,7 @@ namespace NzbDrone.Core.MediaFiles.EpisodeImport
 {
     public interface IDetectSample
     {
-        DetectSampleResult IsSample(Series series, string path, bool isSpecial);
+        DetectSampleResult IsSample(Series series, string path, bool isSpecial, MediaInfoModel mediaInfo = null);
```

Comment:
We need a mediainfo cache by path+size+lastmod. Not now, but it'd eliminate the need for the optional param.